src/cli/log.ts

```diff
@@ -24,10 +24,7 @@ export async function execWithLog<T>(text: string, callback: () => Promise<T>) {
     }
   }, 200);
   try {
-    const output = await callback();
-    return output;
-  } catch (error) {
```

Comment:
same thing here, i think -- `callback` could technically be any `async` function -- if it is a Promise that can reject, this would lead to an `Unhandled Promise Rejection`